modules/bullet/lib/bullet/server.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Bullet.Server do
+  use GenServer
+
+  alias Bullet.Impl, as: Impl
+  def start_link() do
+    start_link(%Bullet{})
+  end
+
```

Comment:
Does a bullet need an absolute time, or could it just have a lifetime that counts down?

Again, just a question.